Parser/parsetok.c

```diff
@@ -322,7 +327,7 @@ parsetok(struct tok_state *tok, grammar *g, int start, perrdetail *err_ret,
             assert(tok->cur - tok->buf < INT_MAX);
             err_ret->offset = (int)(tok->cur - tok->buf);
             len = tok->inp - tok->buf;
-            err_ret->text = (char *) PyObject_MALLOC(len + 1);
+            err_ret->text = (char *) PyObject_Malloc(len + 1);
```

Comment:
PyObject_FREE is used in err_input.

Reply:
I guess that you are asking me to replace PyObject_FREE with PyObject_Free(): done